client/app/services/query-string.js

```diff
@@ -0,0 +1,11 @@
+import qs from 'qs';
```

Comment:
The file name doesn't follow the convention of `filename` = `name of default export`.

Reply:
I was unaware of that convention. This will be handled in #3687